plugins/ftdidmx/FtdiDmxThread.h

```diff
@@ -26,21 +26,53 @@
 #ifndef PLUGINS_FTDIDMX_FTDIDMXTHREAD_H_
 #define PLUGINS_FTDIDMX_FTDIDMXTHREAD_H_
 
+#include <queue>
+#include <utility>
+
 #include "ola/DmxBuffer.h"
 #include "ola/thread/Thread.h"
+#include "ola/rdm/RDMCommand.h"
+#include "ola/rdm/DiscoveryAgent.h"
+#include "ola/rdm/RDMResponseCodes.h"
 
 namespace ola {
 namespace plugin {
 namespace ftdidmx {
 
-class FtdiDmxThread : public ola::thread::Thread {
+enum {
+  HALF_SECOND_MS = 500,
```

Comment:
Why is it 500ms? Is this a time from the spec, if so name it as such, otherwise please name what it's the wait for.